src/image/ExposureInfo.cc

```diff
@@ -254,7 +254,10 @@ void ExposureInfo::_readFits(
         }
         int wcsId = popInt(*metadata, "WCS_ID");
         try {
-            _wcs = archive.get<Wcs>(wcsId);
+            auto archiveWcs = archive.get<Wcs>(wcsId);
+            if (archiveWcs) {
+                _wcs = archiveWcs;
+            }
```

Comment:
It looks like the warning message below this block is incorrect: we should say we're just using the Wcs we got from the header.

By the way, I just refreshed my memory on when that warning is supposed to appear: it's when we couldn't load the object because it's defined in an extension package that's not setup.  If the WCS wasn't saved in the tables at all, we'll just get a null pointer back, and the block above is in play.  If the WCS was saved but it's been corrupted, we'll get an exception that propagates all the way up.